apps/coscrad-frontend/src/components/resources/spatial-features/thumbnail-presenters/spatial-feature-detail.thumbnail.presenter.tsx

```diff
@@ -55,42 +33,32 @@ export const SpatialFeatureDetailThumbnailPresenter = (
 
     const { name, description, imageUrl } = properties;
 
-    const { type: geometryType, coordinates } = geometry;
-
-    const CoordinatesTextPresenter = lookupTable[geometryType];
-
-    if (isNullOrUndefined(CoordinatesTextPresenter)) {
-        throw new Error(
-            `There is no thumbnail presenter registered for coordinates of geometry type: ${geometryType}`
-        );
-    }
+    const { type: geometryType } = geometry;
 
     return (
-        <Card data-testid={id}>
-            <CardContent>
-                <CardHeader title={name} />
-                <img height="100px" src={imageUrl} alt={`Spatial Feature ${id}`} />
-                <br />
-                {description}
-                <br />
-
-                <Divider />
-                <CoordinatesTextPresenter coordinates={coordinates} />
-                <Collapse in={isExpanded}>
-                    <h3>GEOJSON</h3>
-                    <pre>{JSON.stringify(toGeoJSON(spatialFeature), null, 2)}</pre>
-                </Collapse>
-                <Link to={`/${routes.resources.ofType(ResourceType.spatialFeature).detail(id)}`}>
-                    View Spatial Feature
-                </Link>
-            </CardContent>
-            <CardActionArea>
-                <CardActions>
-                    <span aria-label="View Full JSON" onClick={(_) => setIsExpanded(!isExpanded)}>
-                        {isExpanded ? <ExpandLessIcon /> : <ExpandMoreIcon />}
-                    </span>
-                </CardActions>
-            </CardActionArea>
-        </Card>
+        <Grid container spacing={0}>
+            <Grid item xs={3}>
+                <div data-testid={id} />
+                {/* Preview will eventually include images taken from video or photos, etc. */}
+                <StyledPlaceIcon src={imageUrl} alt={`Spatial Feature ${id}`} />
+            </Grid>
+            <Grid item xs={9}>
+                {/* TODO: consider putting a standardized name property on the view models */}
+                <ResourceNamePresenter name={name} variant="h5" />
+                <SinglePropertyPresenter display="Description" value={description} />
+                <SinglePropertyPresenter display="Feature Type" value={geometryType} />
+            </Grid>
+            <Grid item xs={12} container sx={{ justifyContent: 'flex-end' }}>
+                <Box sx={{ pl: 8 }}>
```

Comment:
I have used an escape hatch here.  But I realize that it's not going to work to re-use the thumbnail presenter here, and for spatial features appearing in the connected resources list.  Is there an escape hatch that would allow us to put a custom presenter just for the pop-up?

Reply:
The `DetailPresenter` is injected into the `CoscradLeafletMap` via its props, as it must be since this is an `ICoscradMap` (my way of using an interface when doing functional programming). So the real question is, how can you configure the system to inject a different detail presenter for the map than the connected resources panel.

I see that there is a `spatial-feature-index.container.tsx` where you can inject this. Perhaps you should add an exceptional case to the loop in `index-to-detail-flow-routes.tsx` for `categorizableType===CategorizableType.spatialFeature` where you manually inject a different detail presenter?

Think about how your test coverage does or doesn't save you from possible issues before doing this refactor. This is exactly the kind of refactor where we want the test to protect against regression.